cfgov/templates/wagtailadmin/table_input.html

```diff
@@ -69,9 +69,47 @@
         </div>
     </div>
 </div>
+<div class="field boolean_field widget-checkbox_input">
+    <label for="{{ attrs.id }}-handsontable-sortable">Sortable table</label>
+    <div class="field-content">
+        <div class="input">
+            <input type="checkbox" id="{{ attrs.id }}-handsontable-sortable" name="handsontable-sortable"/>
+            <span class="help">( Enable sortable table functionality )</span>
+        </div>
+    </div>
+</div>
 <br/>
 <div id="{{ attrs.id }}-width_warning" class="table-block-warning">Warning! Your total column width is greater than 100%! Please adjust your column widths to be less than 100% total before continuing.</div>
+
+<table id="{{ attrs.id }}-sortable-input">
```

Comment:
How about putting this below the Fixed Width Column Options to match the order of the checkboxes?